src/components/Modals/Onboarding/OnboardingBasic.vue

```diff
@@ -27,6 +27,24 @@
           >
           <input class="onboarding-input" v-model="lastName" @input="updateBasic()" />
         </div>
+        <div
+          class="onboarding-inputWrapper onboarding-inputWrapper--name onboarding-inputWrapper--description"
+        >
+          <label class="onboarding-label"
+            ><span class="onboarding-subHeader--font"
+              >Entrance Season<span class="onboarding-required-star">*</span>
```

Comment:
small: personally, I think "semester" makes more sense than season, since we usually refer to the "Fall Semester"

Reply:
there are some inconsistencies with "season" vs "semester" throughout the code… I've been thinking and I actually do like using season in the code to distinguish it from the "year" component, but for the user I think "semester" might be clearer.

Reply:
i'm working with the drop-downs and I keep getting mixed up lol

Reply:
Yeah we used to use "semester" or even "type" in the code but @benjamin-shen switched it to season mostly across the board last semester

Reply:
Not sure which I prefer, but just tested switching it to "Semester" and it makes the text wrap, so it might just be best to leave it regardless
![image](https://user-images.githubusercontent.com/25535093/157144430-f4ed64c4-5151-4218-b8b9-a6d23cfbbd24.png)

Reply:
I think it should be called season, and semester should mean (season, year)

Reply:
sounds good! thanks guys!